### ISSUE TITLE
Explicitly state JSON Object Types

### ISSUE THREAD

**Issue by David-Chadwick** [w3c/vc-data-model#123]
In all the examples, the JSON object is not specified. One has to look inside the type parameter to find out what sort of JSON object it is. However, when the object is embedded in another object e.g. credential inside profile, then the object type is specified. I therefore suggest that each object should starts by stating what type of object it is. Thus Example 1 would become
```
credential: {
  "id": "http://dmv.example.gov/credentials/3732",
  "type": ["Credential", "ProofOfAgeCredential"],
  "issuer": "https://dmv.example.gov/issuers/14",
  "issued": "2010-01-01T19:73:24Z",
  "claim": {
    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
    "ageOver": 21
  },
  "proof": { ... }
} 
```
This would have the added advantage that the first value from the type property can be omitted.

and example 17 would become (removing any reference to JSON-LD)

Example 17: A simple verifiable profile in JSON Format
```
profile: {
  "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
  "credential": [{
    "id": "http://dmv.example.gov/credentials/3732",
    "type": ["ProofOfAgeCredential"],
    "issuer": "https://dmv.example.gov/issuers/14",
    "issued": "2010-01-01T19:73:24Z",
    "claim": {
      "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
      "ageOver": 21
    },
.....
```

**Comment by BigBlueHat** [comment]
@David-Chadwick adding that to the example content would make the JSON invalid, so I'd be 👎 on adding `credential:` or `profile:` to either of the examples (or any of the others).

Example code gets copy/pasted frequently, so adding anything that would make it invalid is inhibitive.

**Comment by David-Chadwick** [comment]
@BigBlueHat.  Its not invalid JSON. It might be invalid JSON-LD. However, an alternative solution has been proposed in #149

**Comment by msporny** [comment]
We addressed this issue in PR https://github.com/w3c/vc-data-model/pull/189 ... @David-Chadwick can we close this issue now that the PR is in the core spec?

**Comment by msporny** [comment]
Closing, as I believe we've addressed this issue in PR #189. Re-open if we haven't.
